src/components/profile/ProfileFetch.js

```diff
@@ -0,0 +1,99 @@
+/* eslint-disable camelcase */
+import React, { Fragment } from 'react';
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import profileImage from '../../assets/images/profile.png';
+
+
+function SubContent(profile, className, label) {
+  return (
+    <Fragment>
+      <p className={className}>
+        {' '}
+        <strong style={{ 'font-weight': 'lighter', 'margin-right': '30px' }}>
+          {label}
+        </strong>
+        {profile}
+        {' '}
+      </p>
+    </Fragment>
+  );
+}
+
+function ProfileComponentContent(profile) {
+  console.log(profile.image, 'great');
+  return (
+    <Fragment>
+      <div className="container-fluid">
+        <div className="profile">
+          <hr className="hr-primary" />
+          <h1 className="profiletitle">User Profile</h1>
+          <hr className="hr-primary" />
+          <div className="profile-data" style={{ display: 'flex' }}>
+            <div
+              className="profile-side"
+              style={{
+                display: 'flex', flex: 1, 'flex-direction': 'column', margin: '0 auto',
+              }}
+            >
+              {(profile.image !== 'url') ? (
+                <img
+                  src={profile.image}
+                  alt=""
+                  width="370em"
+                  height="310em"
+                />
+              )
+                : (
```

Comment:
the indentation of this block and the previous block should be the same. Please refactor